config/et_presets.json

```diff
@@ -1,257 +1,166 @@
         "InterlacedMode": "Progressive",
         "Level": "4",
-        "MaxBitRate": "2400",
-        "MaxReferenceFrames": "3",
-        "Profile": "main"
+        "MaxReferenceFrames": "1",
+        "Profile": "baseline"
       },
       "DisplayAspectRatio": "auto",
       "FixedGOP": "true",
-      "FrameRate": "auto",
+      "FrameRate": "30",
       "KeyframesMaxDist": "90",
-      "MaxFrameRate": "29.97",
       "MaxHeight": "720",
       "MaxWidth": "1280",
       "PaddingPolicy": "NoPad",
-      "SizingPolicy": "ShrinkToFit",
-      "Watermarks": [
-        {
-          "HorizontalAlign": "Left",
-          "HorizontalOffset": "10%",
-          "Id": "TopLeft",
-          "MaxHeight": "10%",
-          "MaxWidth": "10%",
-          "Opacity": "100",
-          "SizingPolicy": "ShrinkToFit",
-          "Target": "Content",
-          "VerticalAlign": "Top",
-          "VerticalOffset": "10%"
-        },
-        {
-          "HorizontalAlign": "Right",
-          "HorizontalOffset": "10%",
-          "Id": "TopRight",
-          "MaxHeight": "10%",
-          "MaxWidth": "10%",
-          "Opacity": "100",
-          "SizingPolicy": "ShrinkToFit",
-          "Target": "Content",
-          "VerticalAlign": "Top",
-          "VerticalOffset": "10%"
-        },
-        {
-          "HorizontalAlign": "Left",
-          "HorizontalOffset": "10%",
-          "Id": "BottomLeft",
-          "MaxHeight": "10%",
-          "MaxWidth": "10%",
-          "Opacity": "100",
-          "SizingPolicy": "ShrinkToFit",
-          "Target": "Content",
-          "VerticalAlign": "Bottom",
-          "VerticalOffset": "10%"
-        },
-        {
-          "HorizontalAlign": "Right",
-          "HorizontalOffset": "10%",
-          "Id": "BottomRight",
-          "MaxHeight": "10%",
-          "MaxWidth": "10%",
-          "Opacity": "100",
-          "SizingPolicy": "ShrinkToFit",
-          "Target": "Content",
-          "VerticalAlign": "Bottom",
-          "VerticalOffset": "10%"
-        }
-      ]
+      "SizingPolicy": "Fit"
     }
   },
   {
+    "Audio": {
+      "BitRate": "224",
+      "Channels": "2",
+      "Codec": "AAC",
+      "CodecOptions": {
+        "Profile": "AAC-LC"
+      },
+      "SampleRate": "44100"
+    },
     "Container": "ts",
-    "Name": "HLS 480P",
-    "Description": "HLS 480P",
+    "Description": "HLS for highest network bandwidth",
+    "Name": "HLS_AV_4_0mpbs",
     "Thumbnails": {
-      "Format": "png",
+      "Format": "jpg",
       "Interval": "300",
-      "MaxHeight": "108",
+      "MaxHeight": "144",
       "MaxWidth": "192",
       "PaddingPolicy": "NoPad",
-      "SizingPolicy": "ShrinkToFit"
+      "SizingPolicy": "Fit"
     },
     "Video": {
-      "BitRate": "1200",
+      "BitRate": "4000",
       "Codec": "H.264",
       "CodecOptions": {
-        "BufferSize": "10800",
         "ColorSpaceConversionMode": "None",
         "InterlacedMode": "Progressive",
         "Level": "4",
```

Comment:
This one needs to be Let's standardized on "Level": "4" based on https://en.wikipedia.org/wiki/H.264/MPEG-4_AVC#Levels